src/scripts/records/VideoInfoRecords.js

```diff
@@ -1,18 +1,18 @@
-/* @flow */
-
-import { Record as ImmutableRecord } from 'immutable'
-
-class VideoInfo extends ImmutableRecord({
-  id: null,
-  title: null,
-  description: null,
-  ipfsHash: null,
-  owner: null
-}) {
-  id: ?string
-  title: ?string
-  description: ?string
-  ipfsHash: ?string
-  owner: ?string
-}
-export default VideoInfo
+// /* @flow */
```

Comment:
Why all commented?

Reply:
We have one single VideoRecord now that replaces "UploadRecords" and "VideoInfoRecords". I'll remove the files..